content/en/docs/setup/install/multicluster/index.md

```diff
@@ -620,13 +620,13 @@ Deploy the `helloworld-v1` application to `cluster1`:
 {{< text bash >}}
 $ kubectl apply --context="${CTX_CLUSTER1}" \
     -f @samples/helloworld/helloworld.yaml@ \
-    -l app=helloworld -l version=v1 -n sample
+    -l version=v1 -n sample
 {{< /text >}}
 
 Confirm the `helloworld-v1` pod status:
 
 {{< text bash >}}
-$ kubectl get pod --context="${CTX_CLUSTER1}" -n sample
+$ kubectl get pod --context="${CTX_CLUSTER1}" -n sample -l app=helloworld
```

Comment:
The `-l app=helloworld` isn't really needed since we just created the sample namespace, so we know that the only pod that should be running is the helloworld one. I think it's just a distraction that may confuse some people.

Reply:
Oh, I see, you're not doing things in the test in the exact same order as the doc instructions, so the app label is needed. On the other hand, it is generally best to execute the test instructions in the same order that the doc tells users to run them if you want to really "test the instructions" as documented.

Reply:
In this case, it's a performance optimization. I want to kick off all the deploys first and then check later.